fractions/javaee/undertow/module.conf

```diff
@@ -3,3 +3,4 @@ org.wildfly.extension.undertow
 io.undertow.core
 io.undertow.servlet
 org.ow2.asm
+org.wildfly.swarm.security
```

Comment:
OK, that makes sense, but the `module.conf` addition would still have to be there, correct?